server/src/main/kotlin/org/jetbrains/teamcity/vault/server/VaultBuildStartContextProcessor.kt

```diff
@@ -125,4 +129,7 @@ class VaultBuildStartContextProcessor(private val connector: VaultConnector) : B
         }
     }
 
+    override fun getOrderId() = "HashiCorpVaultPluginBuildStartContextProcessor"
+
+    override fun getConstraint() = PositionConstraint.last()
```

Comment:
While I can understand the goal with the changes to the agent, I'm not comprehending this one. What do we get out of making this one run last on the server-side?

Reply:
Some plugins (like docker plugin) may implement `BuildStartContextProcessor` to add some new parameters. These parameters' values may be vault references. Thus, we must be sure that we're checking `isShouldEnableVaultIntegration` after other plugins set all the parameters. That's what my idea was about.

Reply:
Thanks!